src/reducers/services.js

```diff
@@ -6,15 +6,17 @@ const plugins = (state, log, version) => {
   switch (type) {
 
   case 'add-route-plugin':
-  case 'add-service-plugin': return [ ...state, parsePlugin(content, version) ];
+  case 'add-service-plugin':
```

Comment:
Lots of whitespace changes in this file, purely to help me read it better. We should probably consistently format this whole codebase soon.

Reply:
Once we have this built in a CI server we can add a linter to enforce formatting.